screens/SignInScreen.js

```diff
@@ -0,0 +1,141 @@
+        });
+    } else {
+      setCurrentError({
+        ...currentError,
+        [ChatErrors.signIn.wrongConfirmation]:
+          'Confirmation password not match',
+      });
+    }
+  };
+
+  return (
+    <View>
+      <Card>
+        <Card.Content style={{justifyContent: 'center'}}>
+          <Title>Type Your Data Below</Title>
+          <HelperTextInput
+            theme={{colors: {primary: Colors.secondary}}}
+            selectionColor={Colors.secondary}
+            label="Email"
+            value={email}
+            onChangeText={text => {
+              setEmail(text);
+              setCurrentError({
+                ...currentError,
+                [ChatErrors.signIn.userDisabled]: '',
+                [ChatErrors.signIn.invalidEmail]: '',
+                [ChatErrors.signIn.userNotFound]: '',
+              });
+            }}
+            visible={
+              currentError[ChatErrors.signIn.userDisabled] ||
+              currentError[ChatErrors.signIn.invalidEmail] ||
+              currentError[ChatErrors.signIn.userNotFound]
+            }
+            errorMessage={
+              currentError[ChatErrors.signIn.userDisabled]
+                ? currentError[ChatErrors.signIn.userDisabled]
+                : currentError[ChatErrors.signIn.invalidEmail]
+                ? currentError[ChatErrors.signIn.invalidEmail]
+                : currentError[ChatErrors.signIn.userNotFound]
+            }
+          />
+          <HelperTextInput
+            theme={{colors: {primary: Colors.secondary}}}
+            selectionColor={Colors.secondary}
+            label="Password"
+            secureTextEntry={secureTextEntry}
+            value={password}
+            onChangeText={password => {
+              setPassword(password);
+              setCurrentError({
+                ...currentError,
+                [ChatErrors.signIn.wrongPassword]: '',
+              });
+            }}
+            visible={currentError[ChatErrors.signIn.wrongPassword]}
+            errorMessage={currentError[ChatErrors.signIn.wrongPassword]}
+          />
+          <HelperTextInput
+            theme={{colors: {primary: Colors.secondary}}}
+            selectionColor={Colors.secondary}
+            label="Confirm Password"
+            secureTextEntry={secureTextEntry}
+            value={confPassword}
+            right={buttonIcon}
+            onChangeText={password => {
+              setConfPassword(password);
+              setCurrentError({
+                ...currentError,
+                [ChatErrors.signIn.wrongConfirmation]: '',
+              });
+            }}
+            visible={currentError[ChatErrors.signIn.wrongConfirmation]}
+            errorMessage={currentError[ChatErrors.signIn.wrongConfirmation]}
+            icon={buttonIcon}
+          />
+          <PlatformButton
+            style={styles.button}
+            color={Colors.secondary}
+            platform={Platform.OS}
+            text={'Sign In'}
```

Comment:
Please choose one style for passing strings as props `{'string'}` or `"string"` and follow it though the code.
Eslint should be aware of it and fix in ideal world